mantidproject/mantid
# #26773 Disabled other tabs on muon guis while other process run

**Description of work.**
 Disabled other tabs on muon analysis and frequency domain analysis gui while other process run

**To test:**
1. open muon analysis gui.
2. Drag phase table tab, sequential fitting tab and results tab out.
3. Load MUSR00015189 and notice that the tabs are disabled while data loads and enabled after load is done.
4. Rebin the data and notice the tabs are disabled while rebin algorithm runs and 
enabled after.



Fixes #26773. 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
